packages/crayons-core/src/components/file-uploader-2/file-uploader-2.tsx

```diff
@@ -0,0 +1,891 @@
+import {
+  Component,
+  Prop,
+  Element,
+  Event,
+  EventEmitter,
+  h,
+  Method,
+  Watch,
+} from '@stencil/core';
+import { TranslationController } from '../../global/Translation';
+import { renderHiddenField } from '../../utils';
+import { InitialUploaderFile, UploaderFile } from './file-uploader2-util';
+import { fileDragSVG, fileErrorSVG } from '../../utils/assets';
+
+let fileCount = 0;
+
+@Component({
+  tag: 'fw-file-uploader-2',
+  styleUrl: 'file-uploader-2.scss',
+  shadow: true,
+})
+export class FileUploader {
+  @Element() host!: HTMLElement;
+
+  /**
+   * name - field name
+   */
+  @Prop() name = '';
+
+  /**
+   * text - file uploader text.
+   */
+  // @i18n({ keyName: 'fileUploader2.text' })
+  @Prop({ mutable: true })
+  text;
+
+  /**
+   * description - file uploader description.
+   */
+  // @i18n({ keyName: 'fileUploader2.description' })
+  @Prop({ mutable: true })
+  description;
+
+  @Prop()
+  infoText = '';
+
+  /**
+   * accept - comma separated string. tells us what file formats file uploader should accept.
+   */
+  @Prop() accept = '';
+
+  /**
+   * maxFileSize - maximum file size the file uploader must accept.
+   */
+  @Prop() maxFileSize = 0;
+
+  /**
+   * acceptError - Error message to display when format is invalid.
+   */
+  // @i18n({ keyName: 'fileUploader2.acceptError' })
+  @Prop({ mutable: true })
```

Comment:
why should this prop be mutable?. are we mutating it ?. Kindly check for the props that are not mutable and remove the `mutable:true` property from them.

Reply:
Removed as suggested from here. 

This was done intentionally to enable file uploader as part of fw-form. I agree on the point, seems meaningless here. I will add it in the file-uploader in fw-form PR.